go-app-ussd_higherhealth_healthcheck.js

```diff
@@ -181,18 +182,38 @@ go.app = (function () {
       return new MenuState(name, {
         question: $([
           "The HIGHER HEALTH HealthCheck is your risk assessment tool. Help us by answering a " +
-          "few questions about you and your health.",
+          "few questions.",
           "",
           "Reply"
         ].join("\n")),
-        error: $("This service works best when you select numbers from the list"),
+        error: $("Select a numbers from the list"),
         accept_labels: true,
         choices: [
-          new Choice("state_terms", $("START"))
+          new Choice("state_terms", $("START")),
+          new Choice("state_language", $("CHANGE LANGUAGE"))
         ]
       });
     });
 
+    self.add("state_language", function(name){
+        return new LanguageChoice(name, {
+        question: $([
+          "Select your language",
+          "",
+          "Reply:"
+        ].join("\n")),
+        accept_labels: true,
+        choices: [
+          new Choice("af", $("Afrikaans")),
+          new Choice("en", $("English")),
+          new Choice("st", $("Sotho")),
+          new Choice("xh", $("Xhosa")),
+          new Choice("zu", $("Zulu")),
```

Comment:
Can we use ISO639-3 codes here, instead of ISO639-1. Even though the current languages fit into ISO639-1, not all official South African languages do, so if we ever expand this to all South African languages, then the two letter codes will no longer be sufficient.